core/src/main/java/io/grpc/internal/AbstractServerImplBuilder.java

```diff
@@ -32,7 +32,9 @@
 package io.grpc.internal;
 
 import static com.google.common.base.MoreObjects.firstNonNull;
+import static com.google.common.base.Preconditions.checkNotNull;
 
+import com.google.common.collect.ImmutableList;
```

Comment:
We're still avoiding guava collect for Android. I just got finished removing them. Could we remove the usage here as well, since it isn't too valuable?